torchaudio/models/wav2vec2/model.py

```diff
@@ -1033,6 +1060,7 @@ def hubert_pretrain_large(
     encoder_ff_interm_dropout: float = 0.0,
     encoder_dropout: float = 0.0,
     encoder_layer_drop: float = 0.0,
+    feature_grad_mult=1.0,
```

Comment:
If it's 1.0, gradient is not modified so why not default to `None`?

Reply:
Sure, will change it.